### ISSUE TITLE
Doesn't work too well on mobile

### ISSUE THREAD

**Issue by loftualt** [Goctionni/KinkList#9]
I've had a few partners (and myself) get their preferences messed up in the output when they tried to fill this list out on mobile

Not sure if this is an easily fixable issue, but it's happened to a few people!

**Comment by wimbarelds** [comment]
Could you elaborate on exactly how it messes up the output? Does it show things selected that you didn't? What exactly is happening?

**Comment by Goctionni** [comment]
By preferences messed up in the output, do you mean things selected that the user didn't? I could definitely see a world in which I make this better for desktop and mobile both. The last version now is a while back :)

**Comment by different55** [comment]
Yeah, taps in one area end up selecting boxes in a totally different area. I've even had it happen on desktop in oddly-shaped windows.
